openverse_catalog/dags/providers/provider_api_scripts/museum_victoria.py

```diff
@@ -1,203 +1,175 @@
+            return None
+        images = VictoriaDataIngester._get_images(media_data)
+        if len(images) == 0:
+            return None
+        meta_data = VictoriaDataIngester._get_metadata(data)
+        title = data.get("displayTitle")
+        image_data = {
+            "foreign_landing_url": foreign_landing_url,
+            "title": title,
+            "meta_data": meta_data,
+        }
+
+        record_images = []
+        for image in images:
+            image.update(image_data)
+            record_images.append(image)
+
+        return record_images
+
+    @staticmethod
+    def _get_images(media_data) -> list[ImageDetails]:
+        images = []
+        for media in media_data:
+            if media.get("type") != "image":
                 continue
-            creators = _get_creator(media)
-            image_data.append(
-                {
-                    "image_id": image_id,
-                    "image_url": image_url,
-                    "height": height,
-                    "width": width,
-                    "license_url": license_url,
-                    "creators": creators,
-                }
+            image_id = media.get("id")
+            image_url, height, width, filesize = VictoriaDataIngester._get_image_data(
+                media
             )
-    return image_data
-
-
-def _get_image_data(media):
-    image_url = None
-    height, width = None, None
-    if "large" in media.keys():
-        image_url = media.get("large").get("uri")
-        height = media.get("large").get("height")
-        width = media.get("large").get("width")
-
-    elif "medium" in media.keys():
-        image_url = media.get("medium").get("uri")
-        height = media.get("medium").get("height")
-        width = media.get("medium").get("width")
-
-    elif "small" in media.keys():
-        image_url = media.get("small").get("uri")
-        height = media.get("small").get("height")
-        width = media.get("small").get("width")
-
-    return image_url, height, width
-
-
-def _get_license_url(media):
-    license_url = None
-    license_ = media.get("licence")
-    if license_ is not None:
-        uri = license_.get("uri")
-        if "creativecommons" in uri:
-            license_url = uri
-    return license_url
-
-
-def _get_metadata(obj):
-    metadata = {}
-
-    metadata["datemodified"] = obj.get("dateModified")
-    metadata["category"] = obj.get("category")
-    metadata["description"] = obj.get("physicalDescription")
-
-    keywords = obj.get("keywords")
-    if type(keywords) == list:
-        metadata["keywords"] = ",".join(keywords)
-
-    classifications = obj.get("classifications")
-    if type(classifications) == list:
-        metadata["classifications"] = ",".join(classifications)
-
-    return metadata
+            license_info = VictoriaDataIngester._get_license_info(media)
+            if image_url is None or image_id is None or license_info is None:
```

Comment:
I think this is fine but in case this gets any larger, we could do:
```suggestion
            if any(val is None for val in [image_url, image_id, license_info]):
```